SWLOR.Game.Server/Service/CombatPoint.cs

```diff
@@ -276,24 +295,26 @@ public static void ClearRecentEnemyLevel(uint player)
         /// <param name="player">The player to receiving the point.</param>
         /// <param name="skill">The skill to associate with the point.</param>
         /// <param name="amount">The number of points to add.</param>
-        /// <returns>True if at least one creature is tagged, false otherwise.</returns>
-        public static void AddCombatPointToAllTagged(uint player, SkillType skill, int amount = 1)
+        /// <returns>Number of creatures tagged by the player, -1 if the tracker has expired or is unavailable.</returns>
+        public static int AddCombatPointToAllTagged(uint player, SkillType skill, int amount = 1)
```

Comment:
I think we're violating single responsibility here. I would suggest keeping this as void and instead adding a public method to retrieve the number of creatures tagged by the player. This will result in fewer dependencies should we need to refactor this method in the future.